src/petals/utils/disk_cache.py

```diff
@@ -0,0 +1,5 @@
+import os
+from pathlib import Path
+
+
+DEFAULT_CACHE_DIR = os.getenv("PETALS_CACHE", Path(Path.home(), ".cache", "petals"))
```

Comment:
```suggestion
DEFAULT_CACHE_DIR = os.getenv("PETALS_CACHE", Path.home() / ".cache" / "petals")
```

Reply:
I don't like that this notation doesn't work for all strings and less readable for Python amateurs